tsconfig.json

```diff
@@ -19,7 +19,18 @@
   "compilerOptions": {
     "outDir": "build.assets/.cache/ts",
     "noEmit": true,
-    "types": ["node", "@types/wicg-file-system-access"]
+    "types": ["node", "@types/wicg-file-system-access"],
+    "paths": {
+      "shared/*": ["web/packages/shared/*"],
+      "design/*": ["web/packages/design/src/*"],
+      "design": ["web/packages/design/src/"],
+      "teleport/*": ["web/packages/teleport/src/*"],
+      "teleport": ["web/packages/teleport/src/"],
+      "teleterm/*": ["web/packages/teleterm/src/*"],
+      "e-teleport/*": ["e/web/teleport/src/*"],
+      "gen-proto-js/*": ["gen/proto/js/*"],
```

Comment:
We might as well remove gen-proto-js, it hasn't been used for a while. The last branch it's on is branch/v14.